lsp-augment.el

```diff
@@ -4,7 +4,8 @@
 ;; MIT License - See LICENSE.md for full terms
 
 ;; Author: Roland Dreier <roland.dreier@gmail.com>
-;; Package-Requires: ((emacs "27.1") lsp-mode markdown-mode)
+;; Package-Requires: ((emacs "27.1") (lsp-mode "6.2") (markdown-mode "2.3"))
```

Comment:
Since I was adding the `Version:` below, I figured to add these too.

I added pretty old versions, based on my search of the versions in my `~/.emacs.d/elpa/`.
Again, amend as per your tastes.

Reply:
I wasn't sure what I should put there, since if I look at my installed lsp-mode package, I see:
```
;; Package-Version: 20250625.202
```
and I'm not sure how versions are handled between "6.2" and "2025..." stuff from ELPA.

With that said the lsp-mode I have installed says it requires markdown-mode "2.3" but the markdown-mode I have installed is "20250624.631" so I don't really understand this versioning stuff?